docs/connectors/mysql.asciidoc

```diff
@@ -1397,6 +1413,15 @@ The following _advanced_ configuration properties have good defaults that will w
 The safe default is `true`.
 Skipping should be used only with care as it can lead to data loss or mangling when binlog is processed.
 
+|`binlog.buffer.size` +
+0.7.0 and later
+|0
+|The size of a look-ahead buffer used by the binlog reader. +
+Under specific conditions it is possible that MySQL binlog contains uncomitted data finished by a `ROLLBACK` statement.
+Typical examples are using savepoints or mixing temporary and regular table changes in a single transaction. +
+When a beginning of a transaction is detected then Debezium tries to roll forward the binlog position and find either `COMMIT` or `ROLLBACK` so it can decide whether the changes from  the transaction will be streamed or not. The size of the buffer defines the mximum number of changes in the transaction that Debezium can buffer while searching for transaction boundaries. If the size of transaction is larger than the buffer then Debezium needs to rewind and re-read the events that has not fit into the buffer while streaming. Value `0` disables buffering. +
```

Comment:
s/mximum/maximum/

Also let's try to stick to the "one sentence/thought per line" rule. Easier to track changes down the road as with such very long lines.

Reply:
Maybe add a sentence that this feature is "incubating", i.e. we're waiting for people to provide feedback, but there still may be rough edges.